### PR TITLE
docs: improve `PredictTextSentimentAnalysisSample`

### LINKED ISSUE #7561: docs: `PredictTextSentimentAnalysisSample` uses unsafe JSON serialization strategy
# Description

The [PredictTextSentimentAnalysisSample](https://github.com/googleapis/java-aiplatform/blob/HEAD/samples/snippets/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java) Java code sample used in documentation [here](https://cloud.google.com/vertex-ai/docs/samples/aiplatform-predict-text-sentiment-analysis-sample) has a flaw as is.

If passed an input containing the `"` character like `how much for 24"x24" pizza boxes` it throws an error:
```text
Exception in thread "main" com.google.protobuf.InvalidProtocolBufferException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 1 column 49 path $.content
	at com.google.protobuf.util.JsonFormat$ParserImpl.merge(JsonFormat.java:1347)
	at com.google.protobuf.util.JsonFormat$Parser.merge(JsonFormat.java:477)
	at aiplatform.PredictTextSentimentAnalysisSample.predictTextSentimentAnalysis(PredictTextSentimentAnalysisSample.java:61)
	at aiplatform.PredictTextSentimentAnalysisSample.main(PredictTextSentimentAnalysisSample.java:39)
```

This can be resolved if the input is properly serialized using `com.google.gson` which is already included in the sample's dependencies.

# Additional Context

I was referred here by @kweinmeister as I'm told this is the new house of what used to live at https://github.com/googleapis/java-aiplatform.

This issue is duplicate of one there: https://github.com/googleapis/java-aiplatform/issues/1097

### PR DESCRIPTION
**Pull request by wallind**
# Description

Updates a code sample to have safe serialization of input which may come unsanitized. 

# Additional Context

I was referred here by @kweinmeister as I'm told this is the new house of what used to live at https://github.com/googleapis/java-aiplatform.

This PR is duplicate of one there: https://github.com/googleapis/java-aiplatform/pull/1098

# Github Ticket(s)
Fixes #7561 ☕️

# Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [x] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [x] API's need to be enabled to test (tell us)
- [x] Environment Variables need to be set (ask us to set them)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [x] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Comment by wallind** [comment]
> API's need to be enabled to test (tell us)
> Environment Variables need to be set (ask us to set them)

apparently per the checklist I need to ask for you guys to do these things? I didn't mess with anything remotely complex so I can't see how it applies to me but asking none-the-less.

File: aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java [hunk 1]
```diff
@@ -52,13 +53,15 @@ static void predictTextSentimentAnalysis(String project, String content, String
     try (PredictionServiceClient predictionServiceClient =
         PredictionServiceClient.create(predictionServiceSettings)) {
       String location = "us-central1";
-      String jsonString = "{\"content\": \"" + content + "\"}";
+
+      JsonObject contentJsonObject = new JsonObject();
+      contentJsonObject.addProperty("content", content);
```
Comment (averikitsch) [inline aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:58]: Approving, but can you add a comment or link on why this safe serialization? Thanks!

**Review by averikitsch — APPROVED** [review]

File: aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java (hunk 1 above)
Comment (wallind) [reply aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:58]: @averikitsch I sure can!

I'll be out with my mates watching a hockey game tonight; but I should be able to attend to this tomorrow.

**Review by anguillanneuf — APPROVED** [review]

File: aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java (hunk 1 above)
Comment (kweinmeister) [reply aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:58]: @wallind Your PR is approved and ready to merge, whenever you're ready with this final touch. Thanks!

Comment (wallind) [reply aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:59]: alright well it definitely wasn't _"tomorrow"_ heh; but here we are. I rebased with a comment included this time. Let me know what you think

File: aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java [hunk 2]
```diff
@@ -52,13 +53,16 @@ static void predictTextSentimentAnalysis(String project, String content, String
     try (PredictionServiceClient predictionServiceClient =
         PredictionServiceClient.create(predictionServiceSettings)) {
       String location = "us-central1";
-      String jsonString = "{\"content\": \"" + content + "\"}";
+
+      // Use JsonObject to ensure safe serialization of the input content; handles characters like `"`.
```
Comment (anguillanneuf) [inline aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:57]: Please fix this lint issue:

```
[ERROR] src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:[57] (sizes) LineLength: Line is longer than 100 characters (found 103).
```

Comment (wallind) [reply aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java]: _eek_ sorry. fixed here https://github.com/GoogleCloudPlatform/java-docs-samples/commit/52d3f0929384e9cb6eb5ee3f6f3273edfeaff221

**Comment by wallind** [comment]
thanks for the collaboration guys! _low key_ can't wait for my changes to be rendering on a GCP hosted docs page 👨‍💻 . That _sweet **sweet**_ GCP street cred.

**Comment by kweinmeister** [comment]
Thanks for your contribution, @wallind!

**Pull request merged**